src/audio/pipeline/pipeline-stream.c

```diff
@@ -453,7 +453,9 @@ void pipeline_get_timestamp(struct pipeline *p, struct comp_dev *host,
 	data.start = host;
 	data.posn = posn;
 
-	walk_ctx.comp_func(host, NULL, &walk_ctx, host->direction);
+	if (walk_ctx.comp_func(host, NULL, &walk_ctx, host->direction) !=
+	    PPL_STATUS_PATH_STOP)
+		pipe_warn(p, "pipeline_get_timestamp(): DAI position update failed");
```

Comment:
This has been spamming the logs like crazy for months, see #5017

Reply:
This is non fatal so could be made a dbg trace event.